Src/ReadGps.c

```diff
@@ -6,15 +6,94 @@
 
 #include "Data.h"
 
+
 static int READ_GPS_PERIOD = 1000;
+static int CMD_TIMEOUT = 1500;
+
+static uint8_t header = {0xB5, 0x62};
+static uint8_t CFG_ANT[] = {0xB5, 0x62, 0x06, 0x13, 0x04, 0x00, 0x1B, 0x00, 0x8B, 0xA9}; // Set antenna settings
+static uint8_t CFG_DAT[] = {0xB5, 0x62, 0x06, 0x06, 0x02, 0x00, 0x00, 0x00}; // Set datum settings
+static uint8_t CFG_PRT[];	//TODO: setup configuration for UART
+static uint8_t CFG_RATE[] = {0xB5, 0x62, 0x06, 0x08, 0x06, 0x00, 0xFA, 0x00, 0x01, 0x00, 0x01, 0x00}; // output rate
+static uint8_t CFG_NMEA[]; // not sure if we need to do manually
+static uint8_t setNavigationMode[] = {0xB5, 0x62, 0x06, 0x24, 0x24, 0x00, 0xFF, 0xFF, 0x06, 0x03, 0x00, 0x00, 0x00, 0x00, 0x10, 0x27, 0x00, 0x00, 
```

Comment:
These are globals and have a global namespace, they need to have a very specific name because of that.